performance-tests/src/main/jmeter/disable-job-output.jmx

```diff
@@ -0,0 +1,340 @@
+stepDef.putObject(&quot;options&quot;).put(&quot;outputFormat&quot;, &quot;json&quot;)
+ArtifactService.on(hubClient.getFinalClient()).setArtifact(&quot;stepDefinition&quot;, &quot;jmeterCustomTest&quot;, stepDef)
+
+	
+ObjectNode flow = objectMapper.createObjectNode();
+flow.put(&quot;name&quot;, flowName).putObject(&quot;steps&quot;).putObject(&quot;1&quot;).put(&quot;stepId&quot;, customStep.get(&quot;stepId&quot;).asText());
+ArtifactService.on(hubClient.getFinalClient()).setArtifact(&quot;flow&quot;, flowName, flow);
+
+log.info(&quot;Sending POST request with disableJobOutput: &quot; + disableJobOutputBool)
+</stringProp>
+            </JSR223PreProcessor>
+            <hashTree/>
+          </hashTree>
+        </hashTree>
+        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="HTTP Request: disableJobOutput" enabled="true">
+          <elementProp name="HTTPsampler.Arguments" elementType="Arguments" guiclass="HTTPArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+            <collectionProp name="Arguments.arguments">
+              <elementProp name="rs:flow-name" elementType="HTTPArgument">
+                <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                <stringProp name="Argument.value">jmeterCustomFlow</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+                <boolProp name="HTTPArgument.use_equals">true</boolProp>
+                <stringProp name="Argument.name">rs:flow-name</stringProp>
+              </elementProp>
+              <elementProp name="rs:step" elementType="HTTPArgument">
+                <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                <stringProp name="Argument.value">1</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+                <boolProp name="HTTPArgument.use_equals">true</boolProp>
+                <stringProp name="Argument.name">rs:step</stringProp>
+              </elementProp>
+              <elementProp name="rs:options" elementType="HTTPArgument">
+                <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                <stringProp name="Argument.value">{&quot;disableJobOutput&quot;:${disableJobOutput}}</stringProp>
+                <stringProp name="Argument.metadata">=</stringProp>
+                <boolProp name="HTTPArgument.use_equals">true</boolProp>
+                <stringProp name="Argument.name">rs:options</stringProp>
+              </elementProp>
+            </collectionProp>
+          </elementProp>
+          <stringProp name="HTTPSampler.domain">${host}</stringProp>
+          <stringProp name="HTTPSampler.port">${port}</stringProp>
+          <stringProp name="HTTPSampler.protocol"></stringProp>
+          <stringProp name="HTTPSampler.contentEncoding"></stringProp>
+          <stringProp name="HTTPSampler.path">/v1/resources/mlRunFlow</stringProp>
```

Comment:
We don't want to use this, this endpoint is deprecated (see the comments in it). The code that you have in the "Run flow" component will run the flow, and you can customize that to pass in disableJobOutput in the options Map.